src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -254,15 +255,17 @@ def _upload_resources(clients: UploadClients, upload_state: UploadState) -> None
         media_previously_ingested=upload_state.config.media_previously_uploaded,
     )
 
+    progress_bar = tqdm(upload_state.pending_resources.copy(), leave=False)
```

Comment:
I would vote for leaving the bar after termination. Otherwise, the Terminal history is less understandable, because you see nowhere that the resources have been created. 

Also if an error happens, it might be better understandable if the progress bar is still visible, like so:

```
python -m dsp_tools.cli.entry_point xmlupload testdata/xml-data/test-data-systematic.xml 
You are using DSP-TOOLS version 8.1.0, but version 8.1.0.post18 is available. Consider upgrading via 'pip3 install --upgrade dsp-tools'.

Using DSP server 'http://0.0.0.0:3333' and ingest server 'http://0.0.0.0:3340'
2024-06-18 09:18:56.353154: The XML file is syntactically correct and passed validation.
2024-06-18 09:18:56.923257: Checking resources for circular references...
2024-06-18 09:18:56.982776: Stashing circular references...
Creating Resources (failed: 0):  70%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████                                                      | 40/57 [00:06<00:02,  5.72it/s]

==========================================
Lost connection to DSP server, probably because the server is down. Please continue later with 'resume-xmlupload'. Reason for this failure: Permanently unable to execute the network action. 
                                     Original Message: DuplicateValueException: IRI: \'http://rdfh.ch/4123/5d5d1FKaUC2Wfl4zicggfg\' already exists, try another one.
See logs for more details: /Users/nussbaum/.dsp-tools/logging.log
Saved the current upload state to /Users/nussbaum/.dsp-tools/xmluploads/localhost/resumable/latest.pkl.
```

Reply:
Good point. Should we leave the bar, or should I add a print statement indicating that the resources were uploaded?

Reply:
I would just leave the bar.

Reply:
done (https://github.com/dasch-swiss/dsp-tools/pull/1022/commits/f236e3dfe36685727aa0c5ca41253cce0e1eff23) plus added an initial description for consistency